src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -44,26 +63,62 @@ extern "C" SSL_CTX* CryptoNative_SslCtxCreate(SSL_METHOD* method)
     return ctx;
 }
 
+static uint32_t ParseOpenSSLVersion(const char* string_version)
```

Comment:
This function is new. Should it be more defensive in parsing?

Reply:
Hm.   Being more defensive seems good, since people have been known to bind into libressl.

I think the right answer is to also bind OpenSSL_version_num; but not make the CryptoNative_ export for it.  (So in the caller below you'd just have the inlined version of `uint32_t version = (uint32_t)OpenSSL_version_num();`